src/Sarif/Autogenerated/AnnotatedCodeLocationEqualityComparer.cs

```diff
@@ -0,0 +1,149 @@
+// Copyright (c) Microsoft.  All Rights Reserved.
+// Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.CodeDom.Compiler;
+using System.Collections.Generic;
+
+namespace Microsoft.CodeAnalysis.Sarif
+{
+    /// <summary>
+    /// Defines methods to support the comparison of objects of type AnnotatedCodeLocation for equality.
+    /// </summary>
+    [GeneratedCode("Microsoft.Json.Schema.ToDotNet", "0.20.0.0")]
+    public sealed class AnnotatedCodeLocationEqualityComparer : IEqualityComparer<AnnotatedCodeLocation>
+    {
+        public static readonly AnnotatedCodeLocationEqualityComparer Instance = new AnnotatedCodeLocationEqualityComparer();
+
+        public bool Equals(AnnotatedCodeLocation left, AnnotatedCodeLocation right)
+        {
+            if (ReferenceEquals(left, right))
+            {
+                return true;
+            }
+
+            if (ReferenceEquals(left, null) || ReferenceEquals(right, null))
+            {
+                return false;
+            }
+
+            var physicalLocationEqualityComparer = new PhysicalLocationEqualityComparer();
```

Comment:
I wrote the code to generate this line before I added the `Instance` properties (above), and forgot to change it. Will change to `left.PhysicalLocation.EqualsByValue(right.PhysicalLocation)` or something like that when I introduce that API.